lib/constants.js

```diff
@@ -14,7 +14,10 @@ const schema = yup.object().shape({
   url: yup.string().trim().url().required(),
 });
 
-const db = require('monk')(process.env.MONGODB_URI);
+const db = require('monk')(
+  `${process.env.MONGO_USERNAME}:${process.env.MONGO_PASSWORD}@localhost:${process.env.MONGO_PORT}/lwspw?authSource=admin`
+);
+db.then(() => console.log('Connected!'));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/dChancellor/url_shortener/pullRequest?prid=7437130)
```suggestion
db.then(() => console.log("Connected!"));
```